File: Data Structure/Lab 1/OddEven.c
//WAP to find whether the given number is odd or even

#include<stdio.h>
void main(){
    int num;

    printf("Enter number");
    scanf("%d",&num);

    if(num % 2 == 0){
        printf("Given number is even");
    }
    else{
        printf("Given number is odd");
    }
}

File: Data Structure(Java)/Lab 1/OddEven.java
//WAP to find whether a number is odd or even. 

import java.util.Scanner;

public class OddEven {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter number:");
        int n = sc.nextInt();

        if(n % 2 == 0){
            System.out.println("Given number is even.");
        }
        else{
            System.out.println("Given number is odd.");
        }

        sc.close();
    }
}
